RIOT-OS/RIOT
# make: add blob utility header

### Contribution description

This PR adds a utility makefile that allows arbitrary files to be made available as headers.
This is supposed to be used with, e.g., lua, jerryscript, micropython for scripting language source files.



### Testing procedure

Contains a test application. Check CI output.

Also, try changing the blobs and see if everything gets rebuilt correctly. Hammer this using ```make -j ...```.
Do the same with ```BUILD_IN_DOCKER=1```.




### Issues/PRs references

Simpler alternative to #11497.
